modules/roles_profiles/manifests/profiles/mac_v3_signing.pp

```diff
@@ -10,8 +10,6 @@
             $worker_type  = 'mac-v3-signing'
             $worker_group = regsubst($facts['networking']['fqdn'], '.*\.releng\.(.+)\.mozilla\..*', '\1')
 
-            include puppet::disable_atboot
```

Comment:
Just for my own clarity: we're not using this because these are long running machines instead of ones that reboot per job?

I'm a bit surprised we couldn't have tweaked this module, since it's already a thing that knows how to run puppet, but I suspect you probably either tried it, or had good reasons not to. (I don't think it's worth doing at this point, certainly.)